packages/yarnpkg-libzip/artifacts/build.sh

```diff
@@ -48,6 +48,12 @@ LIBZIP_REPO=arcanis/libzip
   if [[ -n "$LIBZIP_REPO" ]]; then
     if ! [[ -e libzip-"$LIBZIP_VERSION" ]]; then
       git clone https://github.com/"$LIBZIP_REPO" libzip-"$LIBZIP_VERSION"
+
+      # Reverts https://github.com/arcanis/libzip/pull/1
+      # TODO: Remove this once the function is removed from upstream
+      cd libzip-"$LIBZIP_VERSION"
+      git checkout 664462
+      cd ..
```

Comment:
@arcanis Could you remove the commit from the fork?

Reply:
I did, but I feel like it's a good idea nonetheless to pin the commit, so I just removed the comment from this file 🤔